core/src/main/java/org/sonarsource/sonarlint/core/WsHelperImpl.java

```diff
@@ -85,4 +93,62 @@ public String generateAuthenticationToken(ServerConfiguration serverConfig, Stri
     SonarLintWsClient client = createClient(serverConfig);
     return generateAuthenticationToken(new ServerVersionAndStatusChecker(client), client, name, force);
   }
+
+  @Override
+  public List<RemoteOrganization> listOrganizations(ServerConfiguration serverConfig) {
+    SonarLintWsClient client = createClient(serverConfig);
+    ServerVersionAndStatusChecker serverChecker = new ServerVersionAndStatusChecker(client);
+    return listOrganizations(client, serverChecker);
+  }
+
+  static List<RemoteOrganization> listOrganizations(SonarLintWsClient client, ServerVersionAndStatusChecker serverChecker) {
+    try {
+      serverChecker.checkVersionAndStatus("6.4");
+    } catch (UnsupportedServerException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)